NEWS

```diff
@@ -28,6 +28,8 @@ NEWS - user visible changes				-*- outline -*-
    build system do not correctly work together to locate files from
    diagnostic output
 
+** New option -fdefault-file-colseq to specify default file collating sequence
```

Comment:
add  the file COLLATING SEQUENCE in general under "new features" (with the note "currently only with the BDB backend)